2996_reflection/reflection.md

```diff
@@ -4305,27 +4403,79 @@ namespace std::meta {
 :::
 :::
 
-### [meta.reflection.names] Reflection names and locations {-}
+### [meta.reflection.operators] Operator representations
 
 ::: std
 ::: addu
-```cpp
-consteval string_view name_of(info r);
-consteval u8string_view u8name_of(info r);
-```
 
-[#]{.pnum} *Constant When*: If returning `string_view`, the unqualified name is representable using the ordinary string literal encoding. `r` represents a declared entity.
+[#]{.pnum} This enum class specifies constants used to identify operators that can be overloaded, with the meanings listed in Table 1. The values of the constants are distinct.
+
+<center> Table 1: Enum class `operators` [meta.reflection.operators]</center>
+<table  style="text-align:center">
+<tr><th>[Constant]{.addu}</th><th>[Corresponding operator]{.addu}</th></tr>
```

Comment:
Does like... this not work?
```
::: addu
|Constant|Corresponding operator|
|-|-|
|`op_new`|`operator new`|
|`op_delete`|`operator delete`|
...
:::
```

Like as a normal markdown table surrounded by an `addu` fence?

Reply:
Haha seems to work 🤣